packages/react-components/src/organisms/__tests__/TeamCreateOutputForm.test.tsx

```diff
@@ -57,184 +50,242 @@ it('renders the form', async () => {
   expect(getByText(/What are you sharing/i)).toBeVisible();
 });
 
-it('displays current team within the form', async () => {
+it('displays proper message when no author is found', async () => {
+  const getAuthorSuggestions = jest.fn();
+  getAuthorSuggestions.mockResolvedValue([]);
   const { getByText } = render(
     <StaticRouter>
       <TeamCreateOutputForm
         {...props}
-        team={{ ...createTeamResponse(), displayName: 'example team' }}
+        getAuthorSuggestions={getAuthorSuggestions}
       />
     </StaticRouter>,
   );
-  expect(getByText('example team')).toBeVisible();
+  userEvent.click(screen.getByLabelText(/Authors/i));
+  await waitFor(() =>
+    expect(screen.queryByText(/loading/i)).not.toBeInTheDocument(),
+  );
+  expect(getByText(/Sorry, no authors match/i)).toBeVisible();
 });
 
-it('can submit a form when form data is valid', async () => {
-  const saveFn = jest.fn();
+it('displays proper message when no lab is found', async () => {
   const getLabSuggestions = jest.fn();
-  const getAuthorSuggestions = jest.fn();
-  getLabSuggestions.mockResolvedValue([
-    { label: 'One Lab', value: '1' },
-    { label: 'Two Lab', value: '2' },
-  ]);
-  getAuthorSuggestions.mockResolvedValue([
-    {
-      user: { ...createUserResponse(), displayName: 'Chris Blue' },
-      label: 'Chris Blue',
-      value: 'u2',
-    },
-    {
-      user: {
-        ...createUserResponse(),
-        email: undefined,
-        displayName: 'Chris Reed',
-      },
-      label: 'Chris Reed (Non CRN)',
-      value: 'u1',
-    },
-  ]);
-  render(
+  getLabSuggestions.mockResolvedValue([]);
+  const { getByText } = render(
+    <StaticRouter>
+      <TeamCreateOutputForm {...props} getLabSuggestions={getLabSuggestions} />
+    </StaticRouter>,
+  );
+  userEvent.click(screen.getByLabelText(/Labs/i));
+  await waitFor(() =>
+    expect(screen.queryByText(/loading/i)).not.toBeInTheDocument(),
+  );
+  expect(getByText(/Sorry, no labs match/i)).toBeVisible();
```

Comment:
We're using screen on line 81, and not on line 83. Should we be using it consistently?

Reply:
Fixed